ci/build_container/build_container.sh

```diff
@@ -65,6 +65,16 @@ make install
 cd ..
 rm -fr nghttp2*
 
+# c-ares
+wget https://github.com/c-ares/c-ares/archive/cares-1_12_0.tar.gz
```

Comment:
If you use `https://c-ares.haxx.se/download/c-ares-1.12.0.tar.gz` instead you don't need libtool / automake nor `./buildconf`